csstest.js

```diff
@@ -401,6 +404,83 @@ function passclass(info) {
 	return classes[index];
 }
 
+function resetOutput() {
+	mainScore = new Score();
+	_bTestResults = {};
+
+	// Output current score
+	ele('score').textContent = '';
+	ele('passedTests').textContent = '';
+	ele('totalTests').textContent = '';
+	ele('total').textContent = '';
+	ele('specsTested').textContent = '';
+	ele('all').textContent = '';
+
+	// Display time taken
+	ele('timeTaken').textContent = '';
+}
+
+function runTests(filter = '') {
+	let specs = [];
+	let timeBefore = +new Date;
+	let duration = 0;
+
+	let removedWords = / *(?:\([^)]*\)|:.*|\b(?:CSS|Module)\b)( *)/g;
+
+	for (let spec in Specs) {
+		// Filter list of specifications
+		if (filter === 'stable' && Specs[spec]?.status && Specs[spec].status.stability !== 'stable') {
```

Comment:
There is still one occurrence of the "?." operator here.
I have no strong opinion against that if we want to only target really modern browsers now.

Reply:
Oh, sorry! Removed it now.

Sebastian